src/GHCMod.hs

```diff
@@ -256,6 +257,14 @@ globalArgSpec =
                reqArg "OPT" $ \g o ->
                    o { ghcUserOptions = g : ghcUserOptions o }
 
+      , option "" ["map-file"] "Redirect one file to another, --map-file \"file1.hs=file2.hs\"" $
```

Comment:
It'll go into the --help output next time I rewrite the commandline parser, which is soon ;)